contracts/Vouchers.sol

```diff
@@ -12,7 +12,7 @@ import "./interfaces/IVoucherKernel.sol";
 import "./interfaces/ICashier.sol";
 import "./interfaces/IVouchers.sol";
 
-//preparing for ERC-1066, ERC-1444, EIP-838
+//preparing for ERC-1066, ERC-1444
```

Comment:
Please just remove this whole line. We're no planning to implement any of these ERCs.

Reply:
removed.